src/Sarif/Visitors/SarifVersionOneToCurrentVisitor.cs

```diff
@@ -189,7 +191,59 @@ public static Invocation CreateInvocation(InvocationVersionOne v1Invocation, Run
             return invocation;
         }
 
-        public static LogicalLocation CreateLogicalLocation(LogicalLocationVersionOne v1LogicalLocation)
+        internal Location CreateLocation(LocationVersionOne v1Location)
+        {
+            Location location = null;
+
+            if (v1Location != null)
+            {
+                location = new Location
+                {
+                    FullyQualifiedLogicalName = v1Location.LogicalLocationKey ?? v1Location.FullyQualifiedLogicalName,
+                    PhysicalLocation = CreatePhysicalLocation(v1Location.ResultFile),
+                    Properties = v1Location.Properties
+                };
+
+                location.SetProperty($"{FromPropertyBagPrefix}/analysisTarget", v1Location.AnalysisTarget);
+                location.SetProperty($"{FromPropertyBagPrefix}/decoratedName", v1Location.DecoratedName);
```

Comment:
Note to self: find out where `decoratedName` gets put in its new home in the `logicalLocation` object. And since it does have a home there, do we have to put it in the property bag? A v2 => v1 transformer would know where to find it.

Reply:
Likewise, `analysisTarget` goes on the `result`.

Reply:
Yeah I have been of the mind to stash things like this that will be a chore to reassemble.

Reply:
That is a reasonable design decision, and worth a comment.

However -- is there any point in this if we decide (_did_ we decide?) to just paste the entire v1 run into the v2 run's property bag?

Reply:
(This code is out for now, other comment where you pointed out that nothing is calling it)

Reply:
@EasyRhin0 Chris, I think this is still an open issue, right? We're not yet putting `decoratedName` or `analysisTarget` in their new homes.

Reply:
v1 stackFrame doesn't have decoratedName. But I did raise an issue about the logicalLocation pointed to by the key property, see my comment from a few minutes ago.